### DISCUSSION TITLE
Volume based drops

### DISCUSSION THREAD

**Discussion by dhyces** [DynamicTreesTeam/DynamicTrees#688] [Q&A] [answered]
Is there currently a way to make a drop creator with drops that scale with the size of the branch cut?

**Comment by Harleyoc1** [comment, marked as answer]
That is how drops currently work. Whilst this behaviour is not currently possible to change with loot tables, once #706 is merged all DT drops will use loot tables (as such I would not recommend learning the drop creator system right now). 

Below is an example of a 'wood' drop creator for the oak branch. This will be queried for each individual branch block with the resulting items being added into a singular list of item stacks to be dropped at the bottom of the tree after it falls. 

`dynamictrees/loot_tables/trees/wood/oak_branch.json`
```json
{
   "type": "dynamictrees:wood",
   "pools": [
     {
       "rolls": 1,
       "entries": [
         {
           "type": "minecraft:item",
           "functions": [
             {
               "function": "dynamictrees:multiply_logs_count"
             },
             {
               "function": "minecraft:explosion_decay"
             }
           ],
           "name": "minecraft:oak_log"
         }
       ]
     },
     {
       "rolls": 1,
       "entries": [
         {
           "type": "minecraft:item",
           "functions": [
             {
               "function": "dynamictrees:multiply_sticks_count"
             },
             {
               "function": "minecraft:explosion_decay"
             }
           ],
           "name": "minecraft:stick"
         }
       ]
     }
   ]
 } 
```

The `multiply_sticks_count` and `multiply_logs_count` calculate and set the count of the set item stack to drop for sticks and logs respectively. This can then be modified with further vanilla functions, such as explosion decay which is included by default.
